### DISCUSSION TITLE
Dropzone input should accept the default html attributes!

### DISCUSSION THREAD

**Discussion by AdarshJais** [mantinedev/mantine#5874] [Feature requests]
Currently, we're not accepting any attributes for the `input` component ([here](https://github.com/mantinedev/mantine/blob/7b1457f9eb084f0fb663c25793d08877cc957db0/packages/%40mantine/dropzone/src/Dropzone.tsx#L300)) used within the Dropzone component. However, this could be valuable for users requiring customization.

For instance, in my scenario, I have a form system in place where I rely on the `required` attribute to handle validation logic. Situations like this are hindered by the current state of the `Dropzone` component.

_If this is not Intentionally made this way and you guys see this as a valid problem statement, I would love to get my hand dirty with it to contribute_

**Comment by rtivital** [comment]
You are welcome to add `inputProps` prop in a PR

**Reply by AdarshJais** [reply]
Here you go: https://github.com/mantinedev/mantine/pull/5880
